bin/make_db

```diff
@@ -1,5 +1,12 @@
-"""Initialize the dev environment's DB."""
 #!/usr/bin/env python3
-import sys
+"""Initialize the dev environment's DB."""
 from pyramid.paster import bootstrap
-bootstrap("conf/development.ini")
+
+from via.db import create_all, create_engine
+
+
+with bootstrap("conf/development.ini") as env:
+    settings = env["registry"].settings
+
+    engine = create_engine(settings)
+    create_all(engine)
```

Comment:
Made a couple of changes to `bin/make_db` (which I'll have to move into the cookiecutter):

1. You're supposed to use `with bootstrap(...)`, see [Pyramid's docs](https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/commandline.html#writing-a-script)
2. Previously `create_all()` and `stamp()` were [called while bootstrapping the web process](https://github.com/hypothesis/via/blob/35c123986311aa43f6f8e78f43cb31a8f36a4f34/via/db.py#L50-L53). This is why `create_all()` and `stamp()` would be called by the normal web processes when you ran `make dev`. It's also why `bin/make_db` could simply call `bootstrap()` and nothing else. Since I've now changed the normal web process bootstrapping to *not* call `create_all()` or `stamp()`, `bin/make_db` now needs to call them explicitly.

Of course this new code doesn't call `stamp()` yet, it only calls `create_all()`, so I've disabled auto-stamping here. I need to fix that.

Back-porting this into the cookiecutter might be tricky since `bin/make_db` is part of the cookiecutter but `db.py` (where `create_engine()`, `create_all()` and `stamp()` are) is not part of the cookiecutter.